client/src/components/Header/components/language-list.tsx

```diff
@@ -52,11 +52,9 @@ const LanguageList = ({ t, navigate }: LanguageListProps): JSX.Element => {
   // the accessibility tree just needs a little more time to pick up the change.
   // This function allows us to set aria-expanded to false and then delay just a bit before setting focus on the button
   const closeAndFocus = () => {
-    listButtonRef.current?.classList.add('force-show');
     setShowList(false);
     setTimeout(() => {
       listButtonRef.current?.focus();
-      listButtonRef.current?.classList.remove('force-show');
     }, 100);
```

Comment:
>  We need the force-show class to keep the menu open when aria-expanded is set to false otherwise the menu will be display: none and we'll lose keyboard focus before we set it on the button

More context in Bruce's comment https://github.com/freeCodeCamp/freeCodeCamp/pull/50084#discussion_r1175996808

How did you provide the necessary time for the accessibility tree to pick up the change, without the classes?

Reply:
I did not experience any a11y issues using (voiceover) nevertheless the changes have been reverted to avoid any undesired behaviors, these have been added back as the associated css.

Reply:
Not every screen reader may need this fix, and sometimes only one does. It could very well be that NVDA is the only one that needs it.